deploy/helm/templates/config.yaml

```diff
@@ -181,6 +182,7 @@ data:
   TRIVY_DEBUG: {{ .Values.trivy.debug | quote }}
   TRIVY_SKIP_UPDATE: "false"
   TRIVY_DB_REPOSITORY: "{{ .Values.trivy.dbRegistry }}/{{ .Values.trivy.dbRepository }}"
+  TRIVY_JAVA_DB_REPOSITORY: "{{ .Values.trivy.javaDbRegistry }}/{{ .Values.trivy.javaDbRepository }}"
```

Comment:
Not sure its needed as `JAVA_DB_REPOSITORY` is supported [only on client side ](https://github.com/aquasecurity/trivy/issues/3560) for now

Reply:
Right, I added it in the beginning but only later realized that the java db is always pulled by the container doing the scan.